src/shims/foreign_items.rs

```diff
@@ -481,14 +508,15 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
 
             // Architecture-specific shims
             "llvm.x86.sse2.pause" if this.tcx.sess.target.arch == "x86" || this.tcx.sess.target.arch == "x86_64" => {
+                check_abi(abi, Abi::C)?;
```

Comment:
Not sure if this is correct, or should be `Abi::X86Interrupt` (or smth?).

Reply:
Well, the test suite will tell. ;)